float32 average_voltage
float32 average_temperature
float32 maximum_temperature
float32 capacity
